servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/AbstractToFutureTest.java

```diff
@@ -67,6 +71,28 @@ void testSubscribed() {
         assertThat(isSubscribed(), is(true));
     }
 
+    @Test
+    void testCancellableThrows() throws InterruptedException, ExecutionException {
+        doThrow(DELIBERATE_EXCEPTION).when(mockCancellable).cancel();
+        Future<T> future = toFuture();
+        ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
Consider re-using already available `EXEC`

Reply:
Since we are testing `Future` here I didn't want to use our executors which we would also use `Future` to block.